provisioning/install-config-visualizer.sh

```diff
@@ -1,18 +1,13 @@
 #!/usr/bin/env bash
 set -ex
 
-# Get the config-visualizer from GitHub
-if [ ! -d "config-visualizer/" ]; then
-    git clone --depth=1 --branch master https://github.com/precice/config-visualizer.git
-fi
-pip3 install --user -e config-visualizer
+# Install CLI dependencies
+sudo apt-get install -y graphviz
```

Comment:
An alternative could be to install the package from the graphviz repo, or even add it to the VM to keep it reproducible.

Reply:
But it works as-is, right? I want to keep things simple here, while external repositories are sometimes not responding and cancelling the whole VM build (we already have this issue with the OpenFOAM PPA on SourceForge).

I see the reproducibility concern, but I think this reproducible images are not required in the VM.